dragonfly/implementations/daq_run_interface.py

```diff
@@ -101,6 +112,9 @@ def start_run(self, run_name):
         '''
         '''
         self.run_name = run_name
+        if self._run_name is None:
+            logger.critical('run_name is not present; all internal run procedures will be squashed')
+            raise core.exceptions.DriplineValueError('<{}> instance <{}> requires a value for "{}" to initialize run and procedures'.format(self.__class__.__name__, self.name, '_run_name'))
```

Comment:
I guess a raise exception will trigger a logger.critical, which will then be sent to the user on slack.
May be wrong here, this would require a proper test.
But again there is a self.run_name and a self._run_name in the class...